frontend/courses/sample/draw/ellipse.evy

```diff
@@ -0,0 +1,68 @@
+// TODO: stop code scrolling for onkey with execution...
+labels := ["radiusX" "radiusY" "roation°" "start°" "end°"]
+vals := [5 10 45 90 360]
+cnt := len vals
+
+fontfamily "arial"
+textsize 3.5
+cur := 1
+draw_table
+draw_ellipses
+
+on key k:string
+    if k == "ArrowLeft"
+        cur = (cur + cnt - 1) % cnt
+    else if k == "ArrowRight"
+        cur = (cur + 1) % cnt
+    else if k == "ArrowUp"
+        vals[cur] = vals[cur] + 1
+    else if k == "ArrowDown"
+        vals[cur] = vals[cur] - 1
+    end
+    clear
+    draw_ellipses
+    draw_table
+end
+
+func draw_ellipses
+    width 0.5
+    stroke "red"
+    fill "black"
+    ellipse 20 80 vals[0]
+    ellipse 40 80 vals[0] vals[1]
+    ellipse 60 80 vals[0] vals[1] vals[2]
+    ellipse 80 80 vals[0] vals[1] vals[2] vals[3] vals[4]
+
+end
+
+func draw_table
+    fill "none"
+    stroke "black"
+    width 0.2
+    // TODO: broken: for i := range (len labels) // works with labels in local scope
+    for i := range cnt
+        draw_table_column i labels[i] vals[i]
+    end
+    stroke "red"
+    width 0.7
+    // TODO: broken with "" inlined; keeping this as bug report, though param passing not needed
```

Comment:
Minimal sample:
```
fn "s" 0

func fn s:string n:num
    print s n
end
```

not an issue if num is first param. only if empty string is followed

Reply:
ok, fix in https://github.com/foxygoat/evy/pull/118